src/main/java/blusunrize/immersiveengineering/common/IESaveData.java

```diff
@@ -46,7 +50,13 @@ public void readFromNBT(NBTTagCompound nbt)
 				}
 			}
 		}
-
+		
+		Minecraft.getMinecraft().mouseHelper.ungrabMouseCursor();

```

Comment:
Don't merge this just yet, I left an `ungradMouseCursor()` call in that will crash servers.